app/models/Group.js

```diff
@@ -66,6 +67,11 @@ module.exports = function(Sequelize, DataTypes) {
       }
     },
 
+    hostFeePercent: {
+      type: DataTypes.FLOAT,
+      defaultValue: constants.HOST_FEE_PERCENT
```

Comment:
In my view, % isn't a unit (unlike € or $). Percent literally means "* 1/100". So the numeric value for 5% is AFAIK always 0.05, not 5. Storing a value of 5 would be mixing up what the data is (0.05), with its representation ("5 %").